scss/components/_dashboard.scss

```diff
@@ -0,0 +1,103 @@
+@use '../abstracts' as *;
+
+.dashboard {
+    @include flexbox(column, center, center);
+    margin: 1.8rem 0;
+    &__info {
+        align-self: flex-start;
+        width: 80vw;
```

Comment:
why vw not %? You don't need to use vw

Reply:
True, I could just stay with %. I'll change that